google/ggrc-core
# GGRC-1882: Add '--' to Response Options dropdown for CTGOT

# Issue description
Add default value in Response options dropdown for CTGOT.

*Explain the issue that the changes in the pull request solve.*

# Steps to test the changes
Precondition:
Activated one-time WF, a task with Dropdown Task type with values
Steps to reproduce:
1. Go to Active Cycles tab
2. Navigate to Task's Info pane
2. Look at response options dropdown
**Actual Result:** Blank dropdown field is shown
**Expected Result:** Add "--" to response options with dropdown type on Tasks Info pane

# Solution description
Configure dropdown component to show '--' as default value. 

# Sanity checklist


# PR Review checklist
